<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphWindow</class>
 <widget class="QDialog" name="graphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>527</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>535</width>
    <height>527</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>535</width>
    <height>527</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: 
qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FFCDC9, stop:1 #F5B7CA);
</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>491</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  font-size: 15px;
  padding: 1px 2px;
  border: none;
  border-radius: 5px;
  margin: 5px;
  background-color: rgb(100, 70, 201);
  color: #ffffff;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Xmin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>-10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Ymin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>-10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Xmax">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_Ymax">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="text">
    <string>Область определения X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="text">
    <string>Область значений Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="text">
    <string>от</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="text">
    <string>от</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="text">
    <string>до</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="text">
    <string>до</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_step">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 70, 201);
font-family: 'Arial', sans-serif;
font-weight: bold;
background: transparent;</string>
   </property>
   <property name="text">
    <string>Шаг:</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  font-size: 15px;
  padding: 1px 2px;
  border: none;
  border-radius: 5px;
  margin: 5px;
  background-color: rgb(100, 70, 201);
  color: #ffffff;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
   </property>
   <property name="text">
    <string>Перестройка!</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
